src/Raven.Studio/typescript/viewmodels/database/tasks/exportDatabase.ts

```diff
@@ -124,8 +130,28 @@ class exportDatabase extends viewModelBase {
 
             return collections.filter(x => x.toLowerCase().includes(filterLowerCase));
         });
+        
+        const commandEndpointUrl = (db: database) => appUrl.forServer() + appUrl.forDatabaseQuery(db) + endpoints.databases.smuggler.smugglerExport;
```

Comment:
can we extract shared logic into new method and only provide some common escape function ?

Reply:
done